### ISSUE TITLE
Feature Suggestion: highlight the paren `%` would jump to

### ISSUE THREAD

**Issue by chrisgrieser** [utilyre/sentiment.nvim#3]
Hi, thanks for this nice little plugin. 

I found it actually looking for something else, namely a plugin that would highlight the parenthesis a `%` would jump to, somewhat a quickscope but for `%`. While this plugin is already useful, I think also (or alternatively) highlighting the `%` bracket could be quite useful. Just like quickscope helps one to "learn" the `f` and `t` movements, this could help to learn the `%` movement.

**Comment by utilyre** [comment]
Hello!

The `%` operator has this "look ahead" functionality that searches for the _next pair_ and not the outer pair. So it doesn't quite make sense to have them both.

Take this snippet for example:

```
{ // <- sentiment should highlight these pairs
  if true { // imagine that the cursor is at the beginning of this line
    // ...
  } // <- `%` jumps here as it "looks ahead"
}
```

As you can see, `%` and sentiment conflict fundamentally.

**Comment by chrisgrieser** [comment]
Yes, I am aware of the difference, that's why I suggested it as a feature request instead of a bug report

**Comment by utilyre** [comment]
I think a fork would be a great approach, because the base functionality of this feature is pretty much the same but the difference is in how you actually use it (the `sentiment.ui` module).

Nonetheless, I'll keep this open in case somebody wants to open a PR for it (but still a fork is better IMO).
